docs/content/gds/admin/configuration/trtl.en.md

```diff
@@ -1,7 +1,98 @@
 ---
 title: "TrtlDB"
-date: 2022-12-21T10:54:47-06:00
-lastmod: 2022-12-21T10:54:47-06:00
+date: 2022-12-23T12:34:23-06:00
+lastmod: 2022-12-23T12:34:23-06:00
 description: "Configuring the Trtl Database"
 weight: 40
----
\ No newline at end of file
+---
+
+TrtlDb is a globally replicated key-value store that uses anti-entropy for eventually consistent replication that minimizes egress costs, maximizes availability, and is partition tolerant. It exposes unary and streaming APIs for data accesses and is secured via mTLS connections. Its environment variables are prefixed with the `TRTL_` tag. The primary configuration is as follows:
+
+| EnvVar               | Type   | Default | Description                                                                                                      |
+|----------------------|--------|---------|------------------------------------------------------------------------------------------------------------------|
+| TRTL_MAINTENANCE     | bool   | false   | Sets the server to maintenance mode, which will respond to requests with Unavailable except for status requests. |
+| TRTL_BIND_ADDR       | string | :4436   | The IP address and port to bind the trtl grpc server to.                                                         |
+| TRTL_METRICS_ADDR    | string | :7777   | The IP address and port to bind the prometheus metrics http server to.                                           |
+| TRTL_METRICS_ENABLED | bool   | true    | If false, disables the prometheus metrics http server.                                                           |
+| TRTL_LOG_LEVEL       | string | info    | The verbosity of logging, one of trace, debug, info, warn, error, fatal, or panic.                               |
+| TRTL_CONSOLE_LOG     | bool   | false   | If true will print human readable logs instead of JSON logs for machine consumption.                             |
```

Comment:
```suggestion
| TRTL_CONSOLE_LOG     | bool   | false   | If true, will print human readable logs instead of JSON logs for machine consumption.                             |
```